code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,acute)))
+#                                                                       
+# datafile7<-datafile7 %>% mutate(gpled,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,gpled)))
+#                                                                       
+#                                                                       
+#                                                                                                                                          
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp=="All" & 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# datafile7<-datafile7 %>% mutate(acute,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,acute)))
+# 
+# datafile7<-datafile7 %>% mutate(gpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+# 
+# 
+# datafile7<-datafile7 %>% mutate(notgpled,
+#                                 if_else(age_grp!="All" | 
+#                                 reason_grp_high_level%!in%c("All","Health and Social Care Reasons",
+#                                 "Code 9","All Delays excl.Code 9","Patient/Carer/Family-related reasons",0,notgpled)))
+
+#Note: used datafile6 as the creation of datafile7 doesn't add anything new - PMc: check .
+
+### 13. Save out census file ----
+
+write_sav(datafile6,paste0(filepath,"All census data.sav"))
+
+write.xlsx(datafile6,paste0(filepath,"All census data.xlsx"))
+
+table(datafile6$areaname)
+
+
+
+
+
+
+################################################################################
+### 14. bed days data ----
+################################################################################
+
+#Get filespath+main bed days file.SAV
+
+#datafile11<-read_spss(paste0(filepath,"main bed days file.sav"))
+
+
+datafile11<-datafile %>% mutate(Healthboard=substring(Healthboard, 5)) # change nhs_board and remove the 'NHS_'
```

Comment:
See "Style Guide" and "Incremental Data-Frame Names" sections in general feedback.
```suggestion
datafile %>% mutate(Healthboard = substring(Healthboard, 5)) # change nhs_board and remove the 'NHS_'
```